src/module/entities/helper.js

```diff
@@ -6,97 +6,590 @@
  */
 
 // Dependencies
-
+const entityTypesHelper = require(MODULES_BASE_PATH + '/entityTypes/helper')
+// const elasticSearch = require(ROOT_PATH + "/generics/helpers/elasticSearch");
+// const userRolesHelper = require(MODULES_BASE_PATH + "/userRoles/helper");
+const FileStream = require('../../generics/file-stream')
+// const userProfileService = require("../../generics/");
 
 const { v4: uuidv4 } = require('uuid')
 
 // const entitiesHelper = require(MODULES_BASE_PATH + "/entities/helper")
-const _ = require("lodash")
+const _ = require('lodash')
 // const programUsersQueries = require(DB_QUERY_BASE_PATH + "/programUsers")
 
 /**
-    * UserProjectsHelper
-    * @class
-*/
+ * UserProjectsHelper
+ * @class
+ */
 
 module.exports = class UserProjectsHelper {
+	static entityDocuments(findQuery = 'all', fields = 'all', limitingValue = '', skippingValue = '', sortedData = '') {
+		return new Promise(async (resolve, reject) => {
+			try {
+				let queryObject = {}
+
+				if (findQuery != 'all') {
+					queryObject = findQuery
+				}
+				let projectionObject = {}
+
+				if (fields != 'all') {
+					fields.forEach((element) => {
+						projectionObject[element] = 1
+					})
+				}
+
+				let entitiesDocuments
+
+				if (sortedData !== '') {
+					entitiesDocuments = await database.models.entities
+						.find(queryObject, projectionObject)
+						.sort(sortedData)
+						.limit(limitingValue)
+						.skip(skippingValue)
+						.lean()
+				} else {
+					entitiesDocuments = await database.models.entities
+						.find(queryObject, projectionObject)
+						.limit(limitingValue)
+						.skip(skippingValue)
+						.lean()
+				}
+				return resolve(entitiesDocuments)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
+	}
+
+	static details(entityType, entityId) {
+		console.log(entityId, entityType, 'line no 73')
```

Comment:
@MallanagoudaB this api is copied from which service ?

Reply:
ml core only